ERCS/erc-7710.md

```diff
@@ -0,0 +1,177 @@
+---
+eip: 7710
+title: Smart Contract Delegation Interfaces
+description: Minimal interfaces to enable smart contracts to consistently delegate capabilities to other contracts or EOAs.
+author: Ryan McPeck (@McOso), Dan Finlay (@DanFinlay), Rob Dawson (@rojotek), Derek Chiang (@derekchiang)
+discussions-to: https://ethereum-magicians.org/t/towards-more-conversational-wallet-connections-a-proposal-for-the-redeemdelegation-interface/16690
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-05-20
+requires: 1271
```

Comment:
```suggestion
requires: 1271, 7579
```

Line 92 strongly implies that reading 7579 is required to understand the meaning of `mode` and `executionCalldata`.

Reply:
Have added the require into the latest version.